### PR TITLE
Refactor to migrate `media-feature-name-no-unknown` to `findMediaFeatureNames` utility

### PR DESCRIPTION
**Pull request by romainmenke**
> Which issue, if any, is this issue related to?

See : https://github.com/stylelint/stylelint/issues/6848

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: lib/rules/media-feature-name-no-unknown/index.js [hunk 1]
```diff
@@ -52,129 +42,41 @@ const rule = (primary, secondaryOptions) => {
...
-									rangeFeatureOperator.test(prevToken[4].value))
-							) {
-								const [, , startIndex, endIndex, { value: featureName }] = token;
-
-								validateFeatureName(atRule, featureName, startIndex, endIndex);
-							}
-						});
-					}
+			const hasComments = atRule.raws.params && atRule.raws.params.raw;
+			const mediaRule = hasComments ? hasComments : atRule.params;
```
Comment (ybiquitous) [inline lib/rules/media-feature-name-no-unknown/index.js:47]: [suggest] Modern operators can simplify the code like this:

```suggestion
			const mediaRule = atRule.raws.params?.raw ?? atRule.params;
```

**Review by ybiquitous — APPROVED** [review]
Thanks for the pull request! Much cleaner code. LGTM 👍🏼 

I've commented on a trivial refactoring suggestion, but you can merge it.

File: lib/rules/media-feature-name-no-unknown/index.js (hunk 1 above)
Comment (ybiquitous) [reply lib/rules/media-feature-name-no-unknown/index.js:47]: Ah, I forgot it, but we can use `getAtRuleParams()` utility instead: 😅 

https://github.com/stylelint/stylelint/blob/fd78ef7ad01123b0c4b05ad55d9986e3e1e21a01/lib/utils/getAtRuleParams.js#L7-L11

Comment (ybiquitous) [reply lib/rules/media-feature-name-no-unknown/index.js:47]: We also can refactor the util, although it's out of the PR's scope.

```diff
 module.exports = function getAtRuleParams(atRule) {
-	const raws = atRule.raws;
-
-	return (raws.params && raws.params.raw) || atRule.params;
+	return atRule.raws.params?.raw ?? atRule.params;
 };
```

Comment (romainmenke) [reply lib/rules/media-feature-name-no-unknown/index.js]: Awesome!
This utility is exactly what was missing :)

**Pull request merged**